### ISSUE TITLE
Mods.toml issues about displayTest and mods,toml

### ISSUE THREAD

**Issue by SrRapero720** [DemonWav/MinecraftDevIntelliJ#2194]
### Minecraft Development for IntelliJ plugin version

2023.3-1.6.12

### IntelliJ version

IntelliJ IDEA 2023.3.1 (Community Edition) Build #IC-233.11799.300, built on December 11, 2023 Runtime version: 17.0.9+7-b1087.7 amd64 VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o. Windows 11.0 GC: G1 Young Generation, G1 Old Generation Memory: 4086M Cores: 12 Registry:   ide.experimental.ui=true Non-Bundled Plugins:   com.intellij.ideolog (222.2.0.0)   de.femtopedia.diffplugin (2.1.12)   com.intellij.grazie.pro (0.3.269)   monokai-pro (1.10)   com.github.pshirshov.bytecodeeditor (0.2.1)   com.demonwav.minecraft-dev (2023.3-1.6.12) Kotlin: 233.11799.300-IJ

### Operating System

Windows 11

### Target platform

Minecraft Forge

### Description of the bug
There are some small issues on showing erros in mods.toml

- DisplayTest is only available since 41.0.15
  - was backported for 1.18.2 in 40.2.14 on https://github.com/MinecraftForge/MinecraftForge/pull/9775 
  - and is pending for 1.16.5 on https://github.com/MinecraftForge/MinecraftForge/pull/9809
- clientSideOnly option added in 49.0.6 on https://github.com/MinecraftForge/MinecraftForge/pull/9804
- new Forge MDK causes "sintax" errors using `${modid}`

I need to point the fact plugin shows all problems as errors instead of a warning, normally errors are displayed just to point errors like syntax errors or Exceptions, otherwise RuntimeException's like NPE are just warned (and others are intended ignored)

In short: mods.toml errors fits better on a warning instead of an error

**Comment by RedNesto** [comment]
Thanks for the info, I don't usually follow backports 👍 

The clientSideOnly option is currently in the nightly channel (see https://github.com/minecraft-dev/MinecraftDev/commit/d704b7b41f5a573771d84c24d252c2058b2e446c#diff-76b3affa52c4a5eb84523bd9641ff9776d0bdade397a33a412bb983b84831810)
There's no checkbox in the creator but at least the templates has it included commented.

> new Forge MDK causes "sintax" errors using ${modid}

This is what the forge MDK does, I'm trying to follow it closely and it's working AFAIK.
Those are simply placeholders expanded by the processResources task as defined in the build.gradle file.

>I need to point the fact plugin shows all problems as errors instead of a warning, normally errors are displayed just to point errors like syntax errors or Exceptions, otherwise RuntimeException's like NPE are just warned (and others are intended ignored)

Thing is, the inspection reports as errors all the issues that **will** end up as errors at runtime if I'm not mistaken.
You can always change the severity in the inspection settings if this doesn't suit you.
